### PR TITLE
Fix wrong setup for Symfony 3.3

### PR DESCRIPTION
**Pull request by tyx**
Also fix the #45 

Sorry for the CS fix, I was too hurry to move them in a dedicated commit.

### CODE REVIEW

**Comment by chalasr** [comment]
Thanks for the fix, I'm not comfortable with the DI test helpers.
Looks good to me 👍

**Comment by tyx** [comment]
Thanks for your PR, I switched my handlers to private, it made my day \o/

File: tests/Integration/BasicCommandAndBusMappingTest.php [hunk 1]
```diff
@@ -36,7 +35,7 @@ public function testHandleCommandWithInvalidMiddleware()
 commandbus:
     default:
         middleware:
-            - tactician.middleware.validator
+            - tactician.middleware.whatever
```
Comment (tyx) [inline tests/Integration/BasicCommandAndBusMappingTest.php]: to not be redundant with the test in `ValidatorTest`

File: src/DependencyInjection/Compiler/CommandHandlerPass.php [hunk 2]
```diff
@@ -65,8 +65,8 @@ public function process(ContainerBuilder $container)
 
             // Leverage symfony/dependency-injection:^3.3 service locators
             if (class_exists(ServiceLocator::class)) {
-                $this->registerHandlerServiceLocator($container, $locatorServiceId, $handlerMapping);
-                $locatorDefinition = $this->buildLocatorDefinition($handlerMapping, 'tactician.commandbus.'.$busId.'.handler.service_locator', ContainerLocator::class);
+                $this->registerHandlerServiceLocator($container, 'tactician.commandbus.'.$busId.'.handler.service_locator', $handlerMapping);
+                $locatorDefinition = $this->buildLocatorDefinition($handlerMapping, ContainerLocator::class, 'tactician.commandbus.'.$busId.'.handler.service_locator');
```
Comment (tyx) [inline src/DependencyInjection/Compiler/CommandHandlerPass.php]: I guess this serviceId deserves a variable to make reading easier

**Pull request merged**